### PR TITLE
Logs color

### PR DESCRIPTION
**Pull request by scottcwilson**
Shows the logs widget in normal color with an "all's well" icon when there are no logs, only switching to red with the warning icon if logs exist.

### CODE REVIEW

File: includes/library/zencart/DashboardWidget/src/Logs.php [hunk 1]
```diff
@@ -20,6 +20,12 @@
  */
 class Logs extends AbstractWidget
 {
+  public function __construct($widgetKey, $widgetInfo = NULL) 
+  {
+      parent::__construct($widgetKey, $widgetInfo);
+      $count = 0;
```
Comment (drbyte) [inline includes/library/zencart/DashboardWidget/src/Logs.php]: Shouldn't `$count` be `$this->count` ?

And a declaration of the variable as `protected $count;` above the constructor?

**Comment by zcwilt** [comment]
:+1:

**Comment by drbyte** [comment]
👍

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**